systemd/consul-client-conf.json.in

```diff
@@ -1,6 +1,11 @@
 {
   "enable_local_script_checks": true,
   "watches": [
+    {
+      "type": "key",
+      "key": "leader",
+      "args": [ "/opt/seagate/cortx/hare/libexec/elect-rc-leader", "--conf-dir", "TMP_CONF_DIR" ]
```

Comment:
I'm not sure if this is correct. As far as I remember, only server nodes can participate in leader election (I can't find the proof in Consul's docs quickly though, so maybe this is not Consul's restriction rather than our assumptions in the code).

Are you sure that client agents can participate in the election?

Reply:
@knekrasov, this is not consul cluster leader election, that happens separately within Consul servers. This is Hare + Motr internal RC leader election between just Hare + Motr nodes. Earlier we had coupled it with Consul server as we started confds on Consul server nodes (a different problem to solve), now that Hare will not assume it is running on any Consul server nodes, Hare has to elect leader amongst itself.

Reply:
Sorry, I don't quite understand. What will happen if the elected RC leader node runs no confd?

cc @andriytk